src/Algorithms.jl

```diff
@@ -23,4 +25,33 @@ function QFT(n::Int)
     circuit
 end
 
+
+"""
+    QuantumVolume(n, depth)
+
+Generate a Quantum Volume circuit of `n` qubits and `depth` layers. See [1] for more details.
+
+# References
+
+[1] Cross, Andrew W., et al. "Validating quantum computers using randomized model circuits." Physical Review A 100.3 (2019): 032328.
+
+"""
+function QuantumVolume(n::Int, depth::Int)
```

Comment:
Type annotations are not needed.
```suggestion
function QuantumVolume(n, depth)
```

Reply:
Okay!